internal/notifier/bitbucket.go

```diff
@@ -49,7 +50,7 @@ func NewBitbucket(addr string, token string) (*Bitbucket, error) {
 
 	comp = strings.Split(id, "/")
 	if len(comp) != 2 {
-		return nil, errors.New("Invalid bitbucket repository id")
+		return nil, fmt.Errorf("Invalid repository id '%s'", id)
```

Comment:
minor: https://github.com/golang/go/wiki/Errors#errors (few cases of this in the code).